### PR TITLE
unix,fs: remove unused variable

### PR DESCRIPTION
**Pull request by RaisinTen**
Since `copy_file_range_support` doesn't do anything currently, it is
better to get rid of it. The remaining validation is enough to check
whether `copy_file_range` is supported or not.

### CODE REVIEW

**Comment by richardlau** [comment]
> 
> 
> Since `copy_file_range_support` doesn't do anything currently

Isn't it a performance optimisation to avoid calling `uv__fs_copy_file_range` if it is determined that it will never work?

**Comment by cjihrig** [comment]
I think @richardlau is correct.

File: src/unix/fs.c [hunk 1]
```diff
@@ -916,25 +916,16 @@ static ssize_t uv__fs_sendfile(uv_fs_t* req) {
     off = req->off;
 
 #ifdef __linux__
-    {
-      static int copy_file_range_support = 1;
-
-      if (copy_file_range_support) {
```
Comment (RaisinTen) [inline src/unix/fs.c:922]: Since `copy_file_range_support` is set to `1`, wouldn't the part inside the `if` block run anyway?

Comment (santigimeno) [reply src/unix/fs.c:922]: Being `copy_file_range_support` static, it's initialized to `1` only the first time this code block runs. The following times will have whatever value it was set later on. For example if the syscall wasn't supported, `copy_file_range_support` will be set to 0 the next time `uv__fs_sendfile()` is called.

Comment (RaisinTen) [reply src/unix/fs.c:922]: Ah, thanks for telling! Sorry, I overlooked `static`.

**Pull request closed without merging**